README.md

```diff
@@ -89,6 +88,20 @@ See [Safely open apps on your Mac](https://support.apple.com/en-us/HT202491) for
 Depending on your security settings, you may also get a panel asking if you wish to allow it to accept
 network connections. You can click "Deny" as we do not need network access currently nor we do request any.
 
+### Package Manager ###
+
+`Logisim-evolution` is available from a bunch of package managers.  
+Note that these are not mentained by the core developers.
+
+* [Flathub](https://flathub.org/apps/details/com.github.reds.LogisimEvolution) (`flatpak install flathub com.github.reds.LogisimEvolution`)
+* [Snap](https://snapcraft.io/logisim-evolution-snapcraft) (`snap install logisim-evolution-snapcraft`)
+* [Homebrew](https://formulae.brew.sh/cask/logisim-evolution) (`brew install --cask logisim-evolution`)
+* [MacPorts](https://ports.macports.org/port/logisim-evolution/details/) (`port install logisim-evolution`)
+* [Chocolatey](https://community.chocolatey.org/packages/logisim-evolution) (`choco install logisim-evolution`)
+* [winget](https://wingetgui.com/apps?id=Logisim-evolution.Logisim-evolution)
+(`winget install --id=Logisim-evolution.Logisim-evolution  -e`)
+* [Arch User Repository](https://aur.archlinux.org/packages/logisim-evolution)
+
```

Comment:
**Attention:** If you should observe a bug in Logisim-evolution while using one of these packages, first make sure that it can be reproduced with the most recent official packages [provided through this repository](https://github.com/logisim-evolution/logisim-evolution/releases) and ideally the HEAD of our [develop branch](https://github.com/logisim-evolution/logisim-evolution/tree/develop) before [creating an issue](https://github.com/logisim-evolution/logisim-evolution/issues) on the official [Logisim-evolution repository](https://github.com/logisim-evolution/logisim-evolution). Otherwise, report the issue to the package maintainer!